### ISSUE TITLE
WebAccessibility

### ISSUE THREAD

**Issue by andre-dietrich** [LiaScript/LiaScript#50]
There are still some points that need to be optimized in terms of Accessibility:

- [ ] Quizzes
- [ ] Surveys
- [ ] Presentations with updated elements
- [ ] Colors, at least in light-mode
- [ ] Menues
- [ ] Todo-Lists
- [ ] ...

**Comment by KoKoKotlin** [comment]
Colors: Increase contrast of results of quizzes

**Comment by pwab** [comment]
I'm not able to check for a11y with [webaccessibility.com](https://www.webaccessibility.com/) or [WAVE](https://wave.webaim.org) because they seem to only check the loading screen. At least the first checker has a tool which can be integrated into firefox as a plugin which then allows to analyze the course. But I wasn't able to export the findings yet.

Here is the view of the liascript docs analized by [Checka11y.css](https://github.com/jackdomleo7/Checka11y.css-browser-extension):

![grafik](https://user-images.githubusercontent.com/19271185/133455697-c4c76976-040a-4487-925e-e535aab17112.png)

**Comment by andre-dietrich** [comment]
Thank you for the link. Quizzes at least should work fine with screen-reader. When we got the designs, we were not aware of the fact that accessibility is such a big thing and so important. That is why some color - contrast ratios might fail, but the plan so far is to add another theme, which is optimized for visibility also makes use of all accessibility feature in echarts or the applied code-editor ...

@pwab by the way, which browser do you use? Since the unicode-symbols in the table of contents appear darker than in my firefox and chrome ...

**Comment by pwab** [comment]
I'll gladly help testing a11y stuff. Just give a ping if you think the styling is ready for a checkup.

> by the way, which browser do you use? Since the unicode-symbols in the table of contents appear darker than in my firefox and chrome ...

Firefox Developer Edition 93.0b5 (64-Bit)

**Comment by andre-dietrich** [comment]
probably on chrome ... if this is an Windows related issue, or if we have to look to CSS again :see_no_evil: 


![grafik](https://user-images.githubusercontent.com/3089101/133601890-80fe2fa7-f7d6-4275-8ccc-09eac8ca2442.png)

**Comment by pwab** [comment]
Hope this helps (Windows 10):

![liascript_browser_comparison](https://user-images.githubusercontent.com/19271185/133612370-d85b5b80-7de4-4afe-9a3f-d9a1a3822f71.png)

By the way:
What does that symbol mean?

**Comment by andre-dietrich** [comment]
Thank you, that clarifies a lot... I mean the star-symbol... I was not aware of the fact, that it is rendered so differently on Windows ...

* Windows:
  
  ![grafik](https://user-images.githubusercontent.com/3089101/133636459-e55ea8c2-436b-4a15-b742-77deb160059a.png)
  
* Linux:

  ![grafik](https://user-images.githubusercontent.com/3089101/133636889-df13c21c-378e-4aca-932c-153f172804ec.png)

**Comment by mgifford** [comment]
Curious to see that this is still open. Probably it is too general a issue.
